src/resolvers/platform/WindowsSecurity.js

```diff
@@ -62,7 +62,25 @@ export default {
 
   async applications (root, args, context) {
     const device = await kmd('os', context)
-    const apps = await Device.applications(root, args, context)
+    const DEFAULT_REG_PATH = 'HKLM\\\\Software\\\\Microsoft\\\\Windows\\\\CurrentVersion\\\\Uninstall'
```

Comment:
This double-escaping is problematic, but what it took to get this to work for now.

Reply:
updated.